doc/releases/changelog-dev.md

```diff
@@ -47,6 +47,9 @@
 
 * Adds tests, adds no-coverage directives, and removes inaccessible logic to improve code coverage.
   [(#2537)](https://github.com/PennyLaneAI/pennylane/pull/2537)
+  
+* The base class `QubitDevice` now accepts data-types for a statevector.
```

Comment:
It may be worth:
* Phrasing this by highlighting `default.qubit` (and potentially mentioning `QubitDevice`) so that users can related to the change;
* Providing a small snippet that showcases the change and that the underlying statevector is affected.

Reply:
Thanks! Updated